src/main/kotlin/me/bristermitten/devdenbot/extensions/RestActions.kt

```diff
@@ -9,9 +10,11 @@ import kotlin.coroutines.suspendCoroutine
  * @author AlexL
  */
 
-suspend fun <T> RestAction<T>.await(): T
-{
+suspend fun <T> RestAction<T>.await(failure: (Continuation<T>, Throwable) -> Unit = {
+        continuation: Continuation<T>, throwable: Throwable ->
```

Comment:
if we're being really petty the types don't need to be specified here, might be worth changing to make it a bit cleaner

Reply:
Seems to error - `Cannot infer a type for this parameter. Please specify it explicitly.`
Although it's a simple fix by adding <T> i think we should just keep the return type.. idk, for clarity, although I will commit a code cleanup change :P

Reply:
i dont mean the return type, i mean the type of `continuation` and `throwable`